reporting.md

```diff
@@ -0,0 +1,615 @@
+is opportunistic in nature, malicious and misconfigured intermediaries can
+interfere with the successful establishment of suitable encryption, and such
+interference is not always detectable by the receiving server. This document
+provides transparency into failures in the SMTP MTA Strict Transport Security
+policy (TODO: Add ref), negotiation of STARTTLS [@!RFC3207], and the DNS-Based
+Authentication of Named Entities (DANE, [@!RFC6698]).
+
+
+{mainmatter}
+
+# Introduction
+
+The STARTTLS extension to SMTP [@!RFC3207] allows SMTP clients and hosts to
+establish secure SMTP sessions over TLS. The protocol design is based on
+"Opportunistic Security" (OS) [@!RFC7435], which provides interoperability for
+clients that do not support it, but means that any attacker who can delete
+parts of the SMTP session (such as the "250 STARTTLS" response) or who can
+redirect the entire SMTP session (perhaps by overwriting the resolved MX record
+of the delivery domain) can perform such a downgrade or interception attack.
+
+Because such "downgrade attacks" are not necessarily apparent to the receiving
+MTA, this document defines a mechanism for sending domains to report on
+failures at multiple parts of the MTA-to-MTA conversation.
+
+Specifically, this document defines a reporting schema that covers failures 
+in routing, STARTTLS negotiation, and both DANE [@!RFC7671] and STS (TODO: Add
+ref) policy validation errors.
+
+Implementers establish a policy by publishing a TXT record in the DNS which
+instructs comliant sending MTAs to deliver reports of delivery and STARTTLS
+failures in the appropriate format to the specified endpoint.
+
+This document is intended as a companion to the specification for SMTP MTA 
+Strict Transport Security (MTA-STS, TODO: Add ref).
+
+## Terminology
+
+The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**,
+**SHOULD**, **SHOULD NOT**, **RECOMMENDED**, **MAY**, and **OPTIONAL**, when
+they appear in this document, are to be interpreted as described in [@!RFC2119].
+
+We also define the following terms for further use in this document:
+
+* STS Policy: A definition of the expected TLS availability and behavior, as
+  well as the desired actions for a given domain when a sending MTA encounters
+  different results.
+* Policy Domain: The domain against which an STS Policy is defined.
+* Sending MTA: The MTA initiating the delivery of an email message.
+
+# Related Technologies
+
+  * This document is intended as a companion to the specification for SMTP MTA
+    Strict Transport Security (MTA-STS, TODO: Add ref).
+
+  * The Public Key Pinning Extension for HTTP [@!RFC7469] contains a JSON-based
+    definition for reporting individual pin validation failures.
+
+  * The Domain-based Message Authentication, Reporting, and Conformance (DMARC)
+    [@!RFC7489] contains an XML-based reporting format for aggregate and
+    detailed email delivery errors.
+
+
```

Comment:
I know I'm being a dweeb here, but how many newlines do you need between sections anyway? ;)